CONTRIBUTING.md

```diff
@@ -5,3 +5,69 @@ We love improvements to our tools! EDGI has general [guidelines for contributing
 ## Submitting Web Monitoring Issues
 
 Issues that are project-wide, or relate heavily to the interaction between different components, should be added to our [Web Monitoring issue queue](https://github.com/edgi-govdata-archiving/web-monitoring/issues). Component-specific issues should be added to their respective repository.
+
+The following are recommended code styling and best practices for the web-monitoring-ui repository. We also have best practices for related to all the [web-monitoring] project (https://github.com/edgi-govdata-archiving/web-monitoring/blob/master/CONTRIBUTING.md) repo.
+
+## UI - Support
+### Browser Support:
+Last two of every major browser (Chrome, Safari, Firefox, Edge) except IE
+
+### ES6+ allowed everywhere
+ES6 and above is allowed. A feature is allowed if it has been formally published or is a Stage 4 proposal.  Otherwise, it is not allowed.
+
+## UI - Code Style / Best Practices
+
+### CSS Methodologies:
+Recommend BEM as much as possible (as opposed to (OOCSS, SMACSS, ITCSS)
+
+### Organization of React files:
+a) Constructor
```

Comment:
Letters aren’t valid for lists in Markdown—or at least not in Github markdown, since it renders like this:

<img width="918" alt="screen shot 2017-12-28 at 1 25 26 pm" src="https://user-images.githubusercontent.com/74178/34423681-9a0f48ac-ebd2-11e7-99f0-064c128f93b3.png">

Reply:
Switched to numbers.